### PR TITLE
Add Import button on homepage when you don't have any models

### PR DESCRIPTION
**Pull request by a10y**
**Before**

At homepage you'd need to navigate to upper-right corner to click thru to the create new model menu

Screenshot 2023-07-05 at 2 11 25 PM

**After**

Import button embedded right in the homepage, making it one-click to import your first model

Screenshot 2023-07-05 at 2 04 48 PM

### CODE REVIEW

**Review by tarrekshaban — COMMENTED** [review]
Added comment!

File: frontend/src/pages/Home.tsx [hunk 1]
```diff
@@ -62,12 +64,23 @@ export default function Home() {
...
-                      No models were found.
-                      This could be because no models have been registered or the server is offline.
-                      To get started, please register a model.
+                  <div className="mt-2 container">
+                    <p>
+                      You don't seem to have any models!
+                      To get started, please import a model file using either an integrated repository or a file.
                     </p>
+                    <div className="mt-4">
+                      <button
```
[the remark this answers is not in the document]
Comment (a10y) [reply frontend/src/pages/Home.tsx:73]: I agree! But yea not being able to do text + Icon, plus the lack of ability to add hover states prevented me from using Button in this case.

Wanna make a ticket to make sure we transition all of the `button`s to `Button` ?

**Comment by tarrekshaban** [comment]
image
Updated layout for cleaner look on first launch riff on your idea!

**Comment by tarrekshaban** [comment]
image

Updated `<DropdownMenu>` to use `<Button>` as its base.

In the menu bar, the Docs Icon is using <Button> behind the scenes while the Create Icon is using DropdownMenu. They have identical styling due to consolidation of Design System to use the same baseline `<Button>`. We need to expose more of the `<Button>` params for the dropdown at some point but this is a good start.

**Comment by tarrekshaban** [comment]
image

Added "bold" button style for Primary, Default and Danger colors. This is in addition to the "minimal" style which is the default.

**Review by tarrekshaban — APPROVED** [review]

**Pull request merged**